app/org/sagebionetworks/bridge/dynamodb/EmailTemplateMarshaller.java

```diff
@@ -0,0 +1,11 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import org.sagebionetworks.bridge.models.studies.EmailTemplate;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.JsonMarshaller;
+
+/**
+ * Marshalls the EmailTemplate class to JSON when persisting using DynamoDB.
+ */
+public class EmailTemplateMarshaller extends JsonMarshaller<EmailTemplate> {
```

Comment:
Should probably add a comment to the effect of "DynamoMarshaller annotation doesn't allow generic types, so we need to make this class so we can use it in the DynamoMarshaller".

Also, have we tried this with just a JsonMarshaller<Object>? Since the type disappears at runtime anyway.